packages/integrations/image/README.md

```diff
@@ -22,8 +22,20 @@ This integration provides `<Image />` and `<Picture>` components as well as a ba
 
 ## Installation
 
+The default image transformer is based on [Squoosh](https://github.com/GoogleChromeLabs/squoosh) and uses web assembly libraries to support most deployment environments, including those that do not support sharp, such as StackBlitz and Netlify Edge.
```

Comment:
It looks kind of bad, but I believe the canonical spelling of “WebAssembly” is pascal case.

Also wondering about leading with “including those that do not support sharp” — we haven’t mentioned sharp at all at this point, so might be confusing why we care that this thing we haven’t mention is or isn’t supported. Is being more generic for a second unhelpful?

```suggestion
The default image transformer is based on [Squoosh](https://github.com/GoogleChromeLabs/squoosh) and uses WebAssembly libraries to support most deployment environments, including StackBlitz and Netlify Edge, which don’t support all image libraries.
```

Reply:
Or, given we move straight into discussion of sharp, flip things, so in the next section we say where sharp won’t work and here keep it to:

```suggestion
The default image transformer is based on [Squoosh](https://github.com/GoogleChromeLabs/squoosh) and uses WebAssembly libraries to support most deployment environments.
```